.github/workflows/development-ci.yml

```diff
@@ -0,0 +1,37 @@
+name: Development to Development-CI Sync
+
+on:
+  push:
+    branches:
+      - development
+  workflow_dispatch:
+    descriptions: "Manually sync Development to Development-CI"
```

Comment:
It is it a valid yml key `descriptions`? Can't find any [docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions) about it

Reply:
hmmm why did i do that lol